packages/p2p-v2/src/components/Table/Table.tsx

```diff
@@ -67,11 +69,20 @@ const Table = <T,>({
                 ref={tableContainerRef}
                 style={{ height: isDesktop && columns.length > 0 ? `calc(${height}px - 3.6rem)` : '100%' }}
             >
-                {table.getRowModel().rows.map(row => (
-                    <div className='p2p-v2-table__content-row' key={row.id}>
-                        {rowRender(row.original)}
-                    </div>
-                ))}
+                {data && data.length > 0 ? (
+                    table.getRowModel().rows.map(row => (
+                        <div className='p2p-v2-table__content-row' key={row.id}>
+                            {rowRender(row.original)}
+                        </div>
+                    ))
+                ) : (
+                    <Text
+                        className='w-full flex items-center justify-center border-b-[1px] border-solid border-b-[#f2f3f4] p-[1.6rem]'
+                        size='sm'
+                    >
+                        {emptyDataMessage}
```

Comment:
for the empty condition, we have a actionscreen component as of now. and the table component in deriv-com/ui doesn't have this option 😬

Reply:
yeahhhhh i saw we have the action screen but its very different to the empty message I added, if you see in the video above, theres no button, no icon, just a message, and the table header is still there 😅

Reply:
![Screenshot 2024-02-15 at 6 18 04 PM](https://github.com/binary-com/deriv-app/assets/103412909/7facd8b4-76cd-4652-998c-0112c8f22e34)
what we can do for the Table component coming in deriv ui, i can add this option so we can pass it

Reply:
ah.. i think this case is there only for the buy sell table... may you can just add a text component below the table.. even when the table is empty the headers will be shown.. and the text component can show the empty message. or you can update the one in deriv-com/ui..

Reply:
i think it would be good to have the empty text message in the deriv-com/ui cause there are other libraries that have that type of functionality, I'll raise this up with shayan

Reply:
ah.. yeah.. in that case `emptyDataMessage: string;` can be made optional parameter